### PR TITLE
Notification olufemi

### PR DESCRIPTION
**Pull request by fletcher17**
#### What does this PR do?
- Fixed Notification screen

#### Proposed changes
- No change as this is the first fix

#### Additional Info
- This feature displays the medical appointment and doctor in charge of appointment

#### Checklist
- [ ] Unit tests
- [ ] UI Test
- [ ] Comment
- [ ] Documentation

#### Screenshots
Original                        |   Updated
:------------------------------:|:------------------------------:
  MicrosoftTeams-image (20)

### CODE REVIEW

File: app/src/main/res/layout/notification_recyclerview_layout.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                    android:background="@drawable/arrow_background"
+                    app:srcCompat="@drawable/ic_baseline_keyboard_arrow_right_24" />
+            </LinearLayout>
+
+        </LinearLayout>
+
+    </androidx.cardview.widget.CardView>
+
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (darothub) [inline app/src/main/res/layout/notification_recyclerview_layout.xml]: Too many linear layout. Use constraint layout.

File: navigation/src/main/res/navigation/home_navigation.xml [hunk 2]
```diff
@@ -61,6 +61,11 @@
         android:name="ng.com.thewhitecellfoundation.haemcam.ui.nutrition.RecipesFragment"
         android:label="recipes_fragment">
     </fragment>
+    <fragment
+        android:id="@+id/fragmentNotification"
+        android:name="ng.com.thewhitecellfoundation.haemcam.ui.home.NotificationFragment"
+        android:label="recipes_fragment">
```
Comment (darothub) [inline navigation/src/main/res/navigation/home_navigation.xml]: Change the label.

**Review by darothub — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/notification_recyclerview_layout.xml (hunk 1 above)
Comment (fletcher17) [reply app/src/main/res/layout/notification_recyclerview_layout.xml]: Done

File: navigation/src/main/res/navigation/home_navigation.xml (hunk 2 above)
Comment (fletcher17) [reply navigation/src/main/res/navigation/home_navigation.xml]: Done

**Pull request merged**